valuesets/ValueSet-England-RemovalReason.xml

```diff
@@ -0,0 +1,41 @@
+<ValueSet xmlns="http://hl7.org/fhir">
+  <id value="England-RemovalReason" />
+  <url value="https://fhir.nhs.uk/ValueSet/England-RemovalReason" />
+  <version value="1.0.0" />
+  <name value="EnglandRemovalReason" />
+  <title value="England Removal Reason" />
+  <status value="draft" />
+  <date value="2023-07-20" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A valueset used for the removal of an FGM Flag" />
+  <purpose value="It's used to remove the FGM Flag if it's no longer in use" />
+  <copyright value="Copyright © 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the \&quot;License\&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \&quot;AS IS\&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+  <compose>
+    <include>
+      <system value="https://fhir.nhs.uk/CodeSystem/England-FGMRemovalReason" />
+      <version value="1.0.0" />
+    </include>
+  </compose>
+  <expansion>
+    <timestamp value="2021-04-13T12:00:00+00:00" />
```

Comment:
This timestamp looks wrong and not needed so probably best to remove

Reply:
Just update to a recent date to show when (roughly) the expansion was done